docs/_includes/DeveloperGuide/Requirements/02_use_cases.md

```diff
@@ -71,7 +71,17 @@ Actor: New User
 
 ---
 
-**UC05 - Mark and Unmark Expenses**
+**UC05 - List all Person**
+
+**MSS**
+1. A user requests to list all person after searching for a specific person.
```

Comment:
```suggestion
1. A user requests to list all person.
```
I think the `after searching for a specific person` part might be too specific since user can still list all person after listing all person just that visually wont change.